### ISSUE TITLE
Offering a bounty to add features to AndBible

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#2217]
Is this something we could do and promote?
https://bountysource.com/ 

This might encourage others to contribute more.

**Comment by tuomas2** [comment]
Maybe...

**Comment by tuomas2** [comment]
... the problem of course is that we can't really promise that code is merged even if it implements the feature. Bounty should be unlocked only if the feature is really accepted and merged to the codebase.

**Comment by agrogers** [comment]
True. A separate build could make it available to the person who posed the
bounty. Depending on how important that feature is to them they may be
happy to stay on that build for a long time.

**Comment by timbze** [comment]
I think using bounties could work, and could be a good idea. But I don't think the feature should be used outside of team/leader-agreed features.

But anyone could pay anyone else to build whatever feature they want and get their own build, that's not a problem. I just don't see how it would work right to use bounties for that.
